runtimes/eden/src/pallets_governance.rs

```diff
@@ -80,6 +82,8 @@ parameter_types! {
 	pub const MotionDuration: BlockNumber = 2 * constants::DAYS;
 	pub const MaxProposals: u32 = 100;
 	pub const MaxMembers: u32 = 50;
+	pub MaxCollectivesProposalWeight: Weight = Perbill::from_percent(50) * RuntimeBlockWeights::get().max_block;
```

Comment:
- [ ] Verify source

Reply:
https://github.com/paritytech/substrate/pull/13771/files

This is used as a limit inside the code. This is OK.